inputFiles/inducedSeismicity/Quasi-dynamic_2d_base.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" ?>
+<Problem>
+  <Solvers>
+    <ImplicitQuasiDynamicEQ
+      name="QDSolver"
+      targetRegions="{ Domain, Fault }"
+      shearImpedance="4.41"
```

Comment:
what is this thing? and why it is in the solver?

Reply:
It is used to account for the energy of the shear waves as they propagate in the surrounding rock and it captures all inertial effects. It's the key modeling assumption that makes the solver quasi dynamic.  

It's also there  because we have not really decided where to put it yet. It's kind of a rock property but for practical purposes it's better to attach it to the fault but I don't really want it to be part of the friction model. For now it's just a scalar but yeah, technically it could be field that varies cell by cell.